1-exercises/A-accessing-values/exercise3.js

```diff
@@ -22,7 +22,12 @@ let basketballTeam = {
 
 // write code here
 
+let topPlayers = basketballTeam.topPlayers;
+topPlayers.sort();
 
+for (let player of topPlayers) {
+  console.log(player);
+}
```

Comment:
good job, you can also do it with map/forEach function in one line : 
` basketballTeam.topPlayers.sort().forEach/map(player => console.log(player))`